assets/js/steps/check-in-items.js

```diff
@@ -77,6 +77,7 @@ function CheckInItems({ actionHandler }) {
     if (context.machineState.get.items) {
         items = adaptListOfBooksToBanner(context.machineState.get.items);
     }
+    const reservedBookForPrint = items.filter(book => book.status === bookStatus.RESERVED).pop();
```

Comment:
New line before this const

Why only one book?

Reply:
Fordi der skal kun printes dén bog som _lige_ er blevet afleveret som er reserveret.

Reply:
Do you know this is always the last element of the filtered array?